lib/vanagon/driver.rb

```diff
@@ -47,7 +47,7 @@ def load_engine(engine_type, platform, target)
 
     def load_engine_object(engine_type, platform, target)
       require "vanagon/engine/#{engine_type}"
-      Object::const_get("Vanagon::Engine::#{engine_type.capitalize}").new(platform, target)
+      @engine = Object::const_get("Vanagon::Engine::#{engine_type.capitalize}").new(platform, target)
```

Comment:
@McdonaldSeanp this looks like a issue from your PR. The engine wasnt getting setup when I was trying to use it.